service/matching/db_task_manager.go

```diff
@@ -396,3 +409,11 @@ func (d *dbTaskManager) finishTask(
 	}
 	d.taskReader.ackTask(info.TaskId)
 }
+
+func (d *dbTaskManager) backoffDispatch(duration time.Duration) {
+	if d.dispatchBackoffTimer == nil {
```

Comment:
Will make the same change as https://github.com/temporalio/temporal/pull/3195 here